nohup: ignoring input
2025-08-15 14:18:13,590 - database - INFO - Database initialized successfully
2025-08-15 14:18:13,627 - root - INFO - Fake Email Bot ba≈üladƒ±ldƒ±...
2025-08-15 14:18:16,694 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getMe "HTTP/1.1 200 OK"
2025-08-15 14:18:16,786 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/deleteWebhook "HTTP/1.1 200 OK"
2025-08-15 14:18:16,787 - telegram.ext.Application - INFO - Application started
2025-08-15 14:18:27,043 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:30,329 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:30,687 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/sendMessage "HTTP/1.1 200 OK"
2025-08-15 14:18:40,419 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:48,683 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:49,077 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/sendMessage "HTTP/1.1 200 OK"
2025-08-15 14:18:50,816 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:50,981 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/answerCallbackQuery "HTTP/1.1 200 OK"
2025-08-15 14:18:51,125 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/editMessageText "HTTP/1.1 200 OK"
2025-08-15 14:18:54,069 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:54,163 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/answerCallbackQuery "HTTP/1.1 200 OK"
2025-08-15 14:18:54,166 - root - ERROR - Update Update(callback_query=CallbackQuery(chat_instance='-8004993996434374616', data='create_email', from_user=User(first_name='“ö…ô…≥Œ±…≥', id=1143980741, is_bot=False, language_code='tr', username='kenanbabayevofficial'), id='4913359874016011585', message=Message(channel_chat_created=False, chat=Chat(first_name='“ö…ô…≥Œ±…≥', id=1143980741, type=<ChatType.PRIVATE>, username='kenanbabayevofficial'), date=datetime.datetime(2025, 8, 15, 14, 18, 49, tzinfo=datetime.timezone.utc), delete_chat_photo=False, edit_date=datetime.datetime(2025, 8, 15, 14, 18, 51, tzinfo=datetime.timezone.utc), entities=(MessageEntity(length=7, offset=101, type=<MessageEntityType.BOT_COMMAND>),), from_user=User(first_name='Fake Mail', id=6939515424, is_bot=True, username='MailAzeBot'), group_chat_created=False, message_id=694, reply_markup=InlineKeyboardMarkup(inline_keyboard=((InlineKeyboardButton(callback_data='create_email', text='üìß Email Yarat'), InlineKeyboardButton(callback_data='check_inbox', text='üì¨ G…ôl…ôn Qutu')), (InlineKeyboardButton(callback_data='settings', text='‚öôÔ∏è Parametrl…ôr'), InlineKeyboardButton(callback_data='domains', text='üåê Domainl…ôr')), (InlineKeyboardButton(callback_data='clear_all', text='üóëÔ∏è T…ômizl…ô'),))), supergroup_chat_created=False, text='üé≠ Fake Email Bot\n\n10 d…ôqiq…ôlik fake email yaradƒ±cƒ±sƒ±na xo≈ü g…ôlmisiniz!\n\nüìß Yeni email yaratmaq √º√ß√ºn /create yazƒ±n')), update_id=459449755) caused error 'NoneType' object has no attribute 'reply_text'
2025-08-15 14:18:56,226 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:56,320 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/answerCallbackQuery "HTTP/1.1 200 OK"
2025-08-15 14:18:56,324 - root - ERROR - Update Update(callback_query=CallbackQuery(chat_instance='-8004993996434374616', data='check_inbox', from_user=User(first_name='“ö…ô…≥Œ±…≥', id=1143980741, is_bot=False, language_code='tr', username='kenanbabayevofficial'), id='4913359873535182747', message=Message(channel_chat_created=False, chat=Chat(first_name='“ö…ô…≥Œ±…≥', id=1143980741, type=<ChatType.PRIVATE>, username='kenanbabayevofficial'), date=datetime.datetime(2025, 8, 15, 14, 18, 49, tzinfo=datetime.timezone.utc), delete_chat_photo=False, edit_date=datetime.datetime(2025, 8, 15, 14, 18, 51, tzinfo=datetime.timezone.utc), entities=(MessageEntity(length=7, offset=101, type=<MessageEntityType.BOT_COMMAND>),), from_user=User(first_name='Fake Mail', id=6939515424, is_bot=True, username='MailAzeBot'), group_chat_created=False, message_id=694, reply_markup=InlineKeyboardMarkup(inline_keyboard=((InlineKeyboardButton(callback_data='create_email', text='üìß Email Yarat'), InlineKeyboardButton(callback_data='check_inbox', text='üì¨ G…ôl…ôn Qutu')), (InlineKeyboardButton(callback_data='settings', text='‚öôÔ∏è Parametrl…ôr'), InlineKeyboardButton(callback_data='domains', text='üåê Domainl…ôr')), (InlineKeyboardButton(callback_data='clear_all', text='üóëÔ∏è T…ômizl…ô'),))), supergroup_chat_created=False, text='üé≠ Fake Email Bot\n\n10 d…ôqiq…ôlik fake email yaradƒ±cƒ±sƒ±na xo≈ü g…ôlmisiniz!\n\nüìß Yeni email yaratmaq √º√ß√ºn /create yazƒ±n')), update_id=459449756) caused error 'NoneType' object has no attribute 'reply_text'
2025-08-15 14:18:57,223 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:57,321 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/answerCallbackQuery "HTTP/1.1 200 OK"
2025-08-15 14:18:57,324 - root - ERROR - Update Update(callback_query=CallbackQuery(chat_instance='-8004993996434374616', data='settings', from_user=User(first_name='“ö…ô…≥Œ±…≥', id=1143980741, is_bot=False, language_code='tr', username='kenanbabayevofficial'), id='4913359872277414859', message=Message(channel_chat_created=False, chat=Chat(first_name='“ö…ô…≥Œ±…≥', id=1143980741, type=<ChatType.PRIVATE>, username='kenanbabayevofficial'), date=datetime.datetime(2025, 8, 15, 14, 18, 49, tzinfo=datetime.timezone.utc), delete_chat_photo=False, edit_date=datetime.datetime(2025, 8, 15, 14, 18, 51, tzinfo=datetime.timezone.utc), entities=(MessageEntity(length=7, offset=101, type=<MessageEntityType.BOT_COMMAND>),), from_user=User(first_name='Fake Mail', id=6939515424, is_bot=True, username='MailAzeBot'), group_chat_created=False, message_id=694, reply_markup=InlineKeyboardMarkup(inline_keyboard=((InlineKeyboardButton(callback_data='create_email', text='üìß Email Yarat'), InlineKeyboardButton(callback_data='check_inbox', text='üì¨ G…ôl…ôn Qutu')), (InlineKeyboardButton(callback_data='settings', text='‚öôÔ∏è Parametrl…ôr'), InlineKeyboardButton(callback_data='domains', text='üåê Domainl…ôr')), (InlineKeyboardButton(callback_data='clear_all', text='üóëÔ∏è T…ômizl…ô'),))), supergroup_chat_created=False, text='üé≠ Fake Email Bot\n\n10 d…ôqiq…ôlik fake email yaradƒ±cƒ±sƒ±na xo≈ü g…ôlmisiniz!\n\nüìß Yeni email yaratmaq √º√ß√ºn /create yazƒ±n')), update_id=459449757) caused error 'NoneType' object has no attribute 'reply_text'
2025-08-15 14:18:58,470 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:58,606 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/answerCallbackQuery "HTTP/1.1 200 OK"
2025-08-15 14:18:59,042 - email_generator - ERROR - Domains fetch error: 0, message='Attempt to decode JSON with unexpected mimetype: text/html; charset=utf-8', url=URL('https://10minuteemail.com/api/domains')
2025-08-15 14:18:59,042 - root - ERROR - Update Update(callback_query=CallbackQuery(chat_instance='-8004993996434374616', data='domains', from_user=User(first_name='“ö…ô…≥Œ±…≥', id=1143980741, is_bot=False, language_code='tr', username='kenanbabayevofficial'), id='4913359870198705276', message=Message(channel_chat_created=False, chat=Chat(first_name='“ö…ô…≥Œ±…≥', id=1143980741, type=<ChatType.PRIVATE>, username='kenanbabayevofficial'), date=datetime.datetime(2025, 8, 15, 14, 18, 49, tzinfo=datetime.timezone.utc), delete_chat_photo=False, edit_date=datetime.datetime(2025, 8, 15, 14, 18, 51, tzinfo=datetime.timezone.utc), entities=(MessageEntity(length=7, offset=101, type=<MessageEntityType.BOT_COMMAND>),), from_user=User(first_name='Fake Mail', id=6939515424, is_bot=True, username='MailAzeBot'), group_chat_created=False, message_id=694, reply_markup=InlineKeyboardMarkup(inline_keyboard=((InlineKeyboardButton(callback_data='create_email', text='üìß Email Yarat'), InlineKeyboardButton(callback_data='check_inbox', text='üì¨ G…ôl…ôn Qutu')), (InlineKeyboardButton(callback_data='settings', text='‚öôÔ∏è Parametrl…ôr'), InlineKeyboardButton(callback_data='domains', text='üåê Domainl…ôr')), (InlineKeyboardButton(callback_data='clear_all', text='üóëÔ∏è T…ômizl…ô'),))), supergroup_chat_created=False, text='üé≠ Fake Email Bot\n\n10 d…ôqiq…ôlik fake email yaradƒ±cƒ±sƒ±na xo≈ü g…ôlmisiniz!\n\nüìß Yeni email yaratmaq √º√ß√ºn /create yazƒ±n')), update_id=459449758) caused error 'NoneType' object has no attribute 'reply_text'
2025-08-15 14:18:59,394 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:18:59,524 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/answerCallbackQuery "HTTP/1.1 200 OK"
2025-08-15 14:18:59,529 - root - ERROR - Update Update(callback_query=CallbackQuery(chat_instance='-8004993996434374616', data='create_email', from_user=User(first_name='“ö…ô…≥Œ±…≥', id=1143980741, is_bot=False, language_code='tr', username='kenanbabayevofficial'), id='4913359872238636664', message=Message(channel_chat_created=False, chat=Chat(first_name='“ö…ô…≥Œ±…≥', id=1143980741, type=<ChatType.PRIVATE>, username='kenanbabayevofficial'), date=datetime.datetime(2025, 8, 15, 14, 18, 49, tzinfo=datetime.timezone.utc), delete_chat_photo=False, edit_date=datetime.datetime(2025, 8, 15, 14, 18, 51, tzinfo=datetime.timezone.utc), entities=(MessageEntity(length=7, offset=101, type=<MessageEntityType.BOT_COMMAND>),), from_user=User(first_name='Fake Mail', id=6939515424, is_bot=True, username='MailAzeBot'), group_chat_created=False, message_id=694, reply_markup=InlineKeyboardMarkup(inline_keyboard=((InlineKeyboardButton(callback_data='create_email', text='üìß Email Yarat'), InlineKeyboardButton(callback_data='check_inbox', text='üì¨ G…ôl…ôn Qutu')), (InlineKeyboardButton(callback_data='settings', text='‚öôÔ∏è Parametrl…ôr'), InlineKeyboardButton(callback_data='domains', text='üåê Domainl…ôr')), (InlineKeyboardButton(callback_data='clear_all', text='üóëÔ∏è T…ômizl…ô'),))), supergroup_chat_created=False, text='üé≠ Fake Email Bot\n\n10 d…ôqiq…ôlik fake email yaradƒ±cƒ±sƒ±na xo≈ü g…ôlmisiniz!\n\nüìß Yeni email yaratmaq √º√ß√ºn /create yazƒ±n')), update_id=459449759) caused error 'NoneType' object has no attribute 'reply_text'
2025-08-15 14:19:09,481 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:19:19,566 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
2025-08-15 14:19:29,663 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot6939515424:AAHF9CCDytHevonpPX3ld1ntORpN6ICHrXI/getUpdates "HTTP/1.1 200 OK"
